docs/registry.md

```diff
@@ -0,0 +1,118 @@
+ # Registry Support for SOCI
+
+SOCI is supported by most registries. The notable exception to this is Duckerhub, which currently does not support subject fields in image manifests.
```

Comment:
is duckerhub a thing? or is this a dockerhub typo?

Reply:
```suggestion
SOCI is supported by most registries. The notable exception to this is Docker Hub, which currently does not support subject fields in image manifests.
```